Controller/Annotations/View.php

```diff
@@ -47,18 +47,6 @@ class View extends Template
     protected $serializerEnableMaxDepthChecks;
 
     /**
-     * Returns the annotation alias name.
-     *
-     * @return string
-     *
-     * @see Sensio\Bundle\FrameworkExtraBundle\Configuration\ConfigurationInterface
-     */
-    public function getAliasName()
-    {
-        return 'view';
```

Comment:
This was useless IMO and BTW removed a check in SensioFrameworkBundle to not have a ``@View`` annotation and a ``@Template`` annotation at the same time.